psl/psl-core/src/common/preview_features.rs

```diff
@@ -81,7 +81,8 @@ features!(
     ReactNative,
     PrismaSchemaFolder,
     OmitApi,
-    TypedSql
+    TypedSql,
+    StrictUndefinedChecks
```

Comment:
too late for this comment at this point since it's not the only deviation but this list is supposed to be alphabetically sorted